manifests/install.yaml

```diff
@@ -14803,6 +14803,7 @@ metadata:
     app.kubernetes.io/name: argo-rollouts
     app.kubernetes.io/part-of: argo-rollouts
   name: argo-rollouts
+  namespace: argo-rollouts
```

Comment:
While true, it shouldn't effect anything.
I would say this is an upstream `kustomize` bug.

Reply:
Agree with you about kustomize should not blindly adding namespace to cluster scoped objects.

> While true, it shouldn't effect anything.

It actually affects GitOps diffing tools because the field is considered a desired value, but is dropped in the live object. Argo CD annoyingly had to build special handling to ignore differences in `metadata.namespace` for cluster scoped objects because other tools/projects are careless about correct specs, and I feel we should avoid being part of the problem if we can help it.

Reply:
So what would you suggest then?
Run a complex `awk` over it after generating the file?